app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+# from app.players.dwarves.dwarf_warrior import DwarfWarrior
+# from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+# from app.players.elves.elf_ranger import ElfRanger
+# from app.players.elves.druid import Druid
+
+
+def calculate_team_total_rating(team: list["Player"]) -> int:
+    return sum(player.get_rating() for player in team)
```

Comment:
```suggestion
def calculate_team_total_rating(team: list[Player]) -> int:
    return sum(player.get_rating() for player in team)
```

For annotation, you don't need to use the class in quotes; it should be written without them.